src/components/UserInfo/UserInfo.tsx

```diff
@@ -1 +1,10 @@
-export const UserInfo = () => {};
+type Props = {
+  email?: string | undefined,
+  name?: string | undefined;
+};
+
+export const UserInfo: React.FC<Props> = ({ email, name }) => (
+  <a className="UserInfo" href={email}>
```

Comment:
It leads to 404. Use [mailto](https://css-tricks.com/snippets/html/mailto-links/)